demo.md

```diff
@@ -0,0 +1,155 @@
+# **RVIP** [Out The Box]
+**[Version]:** 1.0.0
+
+# **What Is This?**
+This document provides instruction to set-up **RVIP** as a complete Robotic Vision suite.
+
+This is based on the combination of ROS packages and external libraries used for testing purposes.
+
+This **suite** is made of:
+1. OpenGR
+2. darknet_ros
+3. rvip
+
+# **Dependencies**
+### This section lists all third-party softwares used by **RVIP**.
+
+**Click** the hyperlinks for download instructions.
+
+1. [Ubuntu 18.04 (Bionic Beaver)](https://ubuntu.com/download/desktop)
+    - For the operating system
+2. [ROS Morenia Melodic](http://wiki.ros.org/melodic/Installation/Ubuntu)
+    - For the ROS base
+3. OpenGR
+    - For pose estimation. `Refer below for download instructions.`
+4. [OpenCV 3.2.0](http://www.codebind.com/linux-tutorials/install-opencv-3-2-ubuntu-18-04-lts-linux/)
+    - For reading and displaying images
+5. [Point Cloud Library](https://askubuntu.com/questions/1134856/how-to-install-pcl-on-ubuntu-18-04)
+    - For processing 3D information
+6. darknet_ros
+    - For object recognition using only 2D RGB images. `Refer below for download instructions.`
+
+# **Connecting To Camera**
+### This section provides instructions on how you can connect **RVIP** to your ROS-supported **Depth Sensor/Camera**.  
+`Ensure that you have finished setup before doing this!`
+
+1. **Identify** **topics** and **frame_id** which your **Depth Sensor/Camera** is publishing the following information on:
```

Comment:
As `frame_id` is a field in a `std_msgs/Header`, I suggest using verbatim formatting for it:

```suggestion
1. **Identify** **topics** and `frame_id` which your **Depth Sensor/Camera** is publishing the following information on:
```

Same for all later occurrences of `frame_id`.

You may actually want to call it "TF frame" instead, as `frame_id` is just a field in a header. "TF frame" is the generic concept.

Reply:
`frame_id` text formatting updated. Appended with `TF Frame` to reference generic concept.
[commit 62b714a8a4b7c13106ddafb5a4534fe127e0ee42](https://github.com/cardboardcode/rvip-1/commit/62b714a8a4b7c13106ddafb5a4534fe127e0ee42)


Did not directly change `frame_id` to `TF Frame` as i believe for users of low to no programming background would be **more comfortable seeing the exact field name** which they need to modify to make **rvip** work.